app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -6,9 +6,36 @@ class MonthlyKeyMetricsReport < BaseReport
 
     attr_reader :report_date
 
-    def perform(date)
+    def perform(date = Time.zone.today)
       @report_date = date
-      csv_for_email = monthly_key_metrics_report_array
+
+      account_reuse_table = account_reuse_queries.account_reuse_report
+      total_profiles_table = account_reuse_queries.total_identities_report
+      total_users_all_time_table = total_user_queries.total_user_count_report
+
+      upload_to_s3(account_reuse_data, report_name: 'account_reuse')
+      upload_to_s3(total_profiles_data, report_name: 'total_profiles', )
+      upload_to_s3(total_users_all_time_table, report_name: 'total_users_all_time')
```

Comment:
Much of this is actually from Luis's PR; you can view my commits individually if you need to separate them out. I'll rebase this Friday as appropriate.